tap_pardot/client.py

```diff
@@ -19,54 +19,28 @@ def __init__(self, message, response_content):
 
 
 class Client:
-    """Lightweight Client wrapper to allow switching between version 3 and 4 API based
-    on availability, if desired."""
-
     api_version = None
-    api_key = None
-    creds = None
+    access_token = None
+    refresh_token = None
+    client_id = None
+    client_secret = None
+    business_unit_id = None
 
     get_url = "{}/version/{}/do/query"
     describe_url = "{}/version/{}/do/describe"
 
-    def __init__(self, creds):
-        self.creds = creds
+    def __init__(self, business_unit_id, client_id, client_secret, refresh_token, access_token="dummy"):
```

Comment:
dummy

Reply:
yeah i got it from here:
https://github.com/anelendata/PyPardotSF/blob/master/pypardot/client.py#L30
used to trigger refresh token on first api call apparently :-)